packages/libraries/cli/src/base-command.ts

```diff
@@ -53,28 +53,19 @@ export default abstract class BaseCommand<T extends typeof Command> extends Comm
   }
 
   success(...args: any[]) {
-    this.log(colors.green('✔'), ...args);
+    this.log(Texture.success(...args));
```

Comment:
My personal opinion is that this class was performing the abstraction of formatted based on state and that an additional class is probably unnecessary. But if we have more areas beyond the cli commands that will benefit from formatting, then this makes sense.

Reply:
More motivation for this change arrives in #6122 where formatters (json, text) are co-located and completely separate from IO (e.g. `this.log`).